pkgs/development/python-modules/yaspin/default.nix

```diff
@@ -0,0 +1,32 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, termcolor
+}:
+
+buildPythonPackage rec {
+  pname = "yaspin";
+  version = "2.1.0";
+  format = "wheel";
+
+  # It's not straightforward to build from source since they don't include a
+  # setup.py file. The whole thing is built with poetry but adding poetry-core
+  # to nativeBuildInputs doesn't work.
```

Comment:
`format = "pyproject";` should also suffice.

Reply:
Thanks for the pointer! Switched to building from source in https://github.com/NixOS/nixpkgs/pull/138474/commits/ace04c804e08ea402d0146b4f81d3fa94593e1e0